library/core/class.form.php

```diff
@@ -274,29 +274,6 @@ public function captcha() {
             // A plugin handled the captcha so don't display anything more.
```

Comment:
At this point there's no purpose to the entire `$handled` construct, right? This just exists to fire an event, now.